notebooks/examples/urth-core-function.ipynb

```diff
@@ -357,6 +359,7 @@
     "        np.arange(0, 10, 0.1), \n",
     "        [y + np.random.normal(scale=1e-10+float(noise)) for y in np.arange(0, 10, 0.1)]\n",
     "    )\n",
+    "    plt.close()\n",
```

Comment:
It would work either way; without the `fig` argument, it will close the current figure, which was created above http://matplotlib.org/api/pyplot_api.html#matplotlib.pyplot.close